test/CMakeLists.txt

```diff
@@ -52,16 +52,46 @@ macro (ADD_QFIELD_QML_TEST TESTNAME TESTSRC)
   get_target_property(QT_TARGET_TYPE ${QT_PKG}::Core TYPE)
   if(${QT_TARGET_TYPE} STREQUAL "STATIC_LIBRARY")
     include("${CMAKE_SOURCE_DIR}/cmake/QtQmlImportScanner.cmake")
+
     file(GLOB test_qmlfiles "${CMAKE_CURRENT_SOURCE_DIR}/qml/*.qml")
     file(GLOB qfield_qmlfiles "${CMAKE_SOURCE_DIR}/src/qml/*.qml")
+    file(GLOB qfield_qmlfiles_geometryeditors
```

Comment:
We need to bring this into qml/cmakelists.txt if we don't want to forget to update one or the other in the future.

Reply:
@m-kuhn , can I just relocate the whole code block into src/qml/CMakeLists.txt ?

Reply:
Ignore, I understand what you want of me sir ;-)